### PR TITLE
Feature: Added settings for default order directions

### LINKED ISSUE #11053: Feature: Add option to change the default sorting order
### What feature or improvement do you think would benefit Files?

Right now, I can only see the default sorting option (for e.g. Date Modified). But there is not option to change the order (Ascending or Descending). Could you please incorporate this feature?

### Requirements

Include an option for default sorting order

### Files Version

```yml
Version: 2.4.21.0
OS Version: 10.0.22621.1105
```

### Windows Version
```yml
Edition:	Windows 11 Pro 
Version:	22H2 
OS build:	22621.1105 
Experience:	Windows Feature Experience Pack 1000.22638.1000.0
```

### Comments

_No response_

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #11053 

**Validation**
How did you test these changes?
- [X] Built and ran the app
- [X] Tested the changes for accessibility

**Screenshots**
image

### CODE REVIEW

File: src/Files.App/ViewModels/SettingsViewModels/FoldersViewModel.cs [hunk 1]
```diff
@@ -250,6 +250,35 @@ public bool OpenFoldersNewTab
 			}
 		}
 
+		public bool SortInDescendingOrder
+		{
+			get => UserSettingsService.LayoutSettingsService.DefaultDirectorySortDirection == SortDirection.Descending;
+			set
+			{
+				if (value != (UserSettingsService.LayoutSettingsService.DefaultDirectorySortDirection == SortDirection.Descending))
```
Comment (yaira2) [inline src/Files.App/ViewModels/SettingsViewModels/FoldersViewModel.cs:258]: Can this be moved from `LayoutSettingsService` to `FoldersSettingsService`?

Comment (hishitetsu) [reply src/Files.App/ViewModels/SettingsViewModels/FoldersViewModel.cs]: Moved.

File: src/Files.Backend/Services/Settings/IFoldersSettingsService.cs [hunk 2]
```diff
@@ -156,6 +156,21 @@ public interface IFoldersSettingsService : IBaseSettingsService, INotifyProperty
...
+		/// </summary>
+		SortDirection DefaultDirectorySortDirection { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value indicating the default grouping direction.
+		/// </summary>
+		SortDirection DefaultDirectoryGroupDirection { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value indicating if directories should be sorted alongside files by default.
```
Comment (yaira2) [inline src/Files.Backend/Services/Settings/IFoldersSettingsService.cs:170]:
```suggestion
		/// Gets or sets a value indicating if directories should be sorted alongside files by.
```

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**